### ISSUE TITLE
Ledger Generate Account Fails, missing dependency

### ISSUE THREAD

**Issue by TrevorJTClarke** [near/near-cli#695]
***Describe the bug***
 Im unable to generate a new account with ledger, appears error from missing dep.

***To Reproduce***
 1\. using NEAR Cli (v1.5.4)
 2\. Command: `near generate-key --useLedgerKey name.near`
 3\. Get error

***Expected behavior***
 I should be able to generate a keypair and approve on a ledger

***Additional context***

```
 Error: Cannot find module '@ledgerhq/hw-transport-node-hid'
 Require stack:

*   /.nvm/versions/node/v15.3.0/lib/node_modules/near-cli/middleware/ledger.js
     ```

**Comment by mikedotexe** [comment]
Are you trying to add a key? It is rather confusing, but I believe you'll want to use `add-key` instead:
https://docs.near.org/docs/tools/near-cli#near-add-key

**Comment by mikedotexe** [comment]
I've also tried installing near-cli today on a fresh new Mac M1. Node 16.6.0, getting the same error.

**Comment by mikedotexe** [comment]
Solved this by using NodeJS 14.20.0
